numismatic/feeds/bravenewcoin.py

```diff
@@ -0,0 +1,46 @@
+from itertools import product
+import attr
+from .base import Feed
+from ..libs.config import get_config
+
+class BraveNewCoin(Feed):
+    config = get_config()['bravenewcoin']
```

Comment:
I think it makes more sense to obtain config information at this point as opposed to getting it in the cli and then passing the argument to the object. Not only is it neater, but there is less "magic" - I can see exactly how I get things over here